spiceaidocs/docs/clients/grafana/index.md

```diff
@@ -6,7 +6,7 @@ pagination_prev: 'clients/index'
 pagination_next: null
 ---
 
-The Spice runtime has a Prometheus endpoint configured by default on port `9000`. There's also prebuilt Grafana dashboards for out of the box monitoring of all your Spice instances.
+The Spice runtime has a Prometheus endpoint configured by default on port `9090`. There's also prebuilt Grafana dashboards for out of the box monitoring of all your Spice instances.
```

Comment:
Technically its not configured by default if you run `spiced` directly. It's only configured by default if you launch the runtime from the CLI via `spice run`

Reply:
We really need an entire dedicated monitoring section.

Reply:
The document will be updated with the language similar to monitoring with Datadog for consistency.  Monitoring section can come in a separate PR.